src/Services/SmartProxy.js

```diff
@@ -25,11 +25,11 @@ export const SmartProxyApi = createApi({
     }),
     getClusters: builder.query({
       query: () => `v2/clusters`,
-    }),
-    getDisplayName: builder.query({
-      query: ({ id, includeDisabled }) =>
-        `v2/cluster/${id}/reports?get_disabled=${includeDisabled}`,
-      transformResponse: (response) => response?.report?.meta?.cluster_name,
+      transformResponse: (response) => {
```

Comment:
Nit-pick: you can omit the usage of curly brackets here since the function consists only of the return statement.